src/test/java/Homework25.java

```diff
@@ -0,0 +1,16 @@
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class Homework25 extends BaseTest {
+    @Test
+    public void deletePlaylist() {
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new  HomePage(getDriver());
+        loginPage.logIn();
+        homePage.enterPlayList();
+        homePage.clickDelete();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

        Assert.assertEquals(newName, getUsersNameOnHeader());

        Assert.assertEquals(newName, getUsernameProfileField());    
 }
```